UnitySetup/UnitySetup.psm1

```diff
@@ -2132,3 +2130,537 @@ function Get-UnityLicense {
         Write-Warning "Alias $($_.Name) already configured by $($alias.Source)"
     }
 }
+
+function Import-UPMConfig {
```

Comment:
For what this method is doing it's importing the manifest, I'd think this would be `Import-ProjectManifest`. `Import-UPMConfig` would be for importing the .upmconfig.toml file.

Reply:
Sure, though I should caveat that this method imports the scoped registries from however many manifests it finds.  By default it's a 3-directory deep scan.

Anyway per the behavior needed for the auth configuration we'll need to return an array of registries and name it `ImportProjectManifests` which I think will work just fine as well

Reply:
Right, I think we want to support importing Project Manifests as well as UpmConfigs.